src/components/Connections.tsx

```diff
@@ -62,16 +71,19 @@ function Connections() {
     y: number;
   } | null>(null); // useful for coordinates
   const [knowledgeStatementsMap, setKnowledgeStatementsMap] =
```

Comment:
Just a minor comment, suggestion for the futures @D-GopalKrishna  @afonsobspinto , if these objects are not of the type Map I would avoid calling them map since I was relying on their API originally when developing the csv logic but they are not.